nncf/common/graph/graph.py

```diff
@@ -700,3 +702,16 @@ def remove_nodes_from(self, nodes: List[NNCFNode]) -> None:
         self._node_id_to_key_dict = {}
         for node_key, node in self._nx_graph.nodes.items():
             self._node_id_to_key_dict[node["id"]] = node_key
+
+    def find_patterns_nodes(self, patterns: GraphPattern) -> List[NNCFNode]:
+        """
+        Returns nodes of matched pattern in patterns.
+
+        :param patterns: Instance of GraphPattern containing all patterns.
+        :return: Nodes that are matched patterns.
```

Comment:
In what order are the nodes returned?

Reply:
The order of the returned nodes relies on DiGraphMatcher logic from networkX. 
As far as I know, the DiGraphMatched does not guarantee a specific order for returning isomorphic subgraphs.

Reply:
Mention that in the parameter description.

Reply:
Done